tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/work_chooser_block.yaml

```diff
@@ -1,33 +1,30 @@
 context:
   value:
     featured_work_heading: Case studies
-    work_pages:
-      - title: Imagining and designing a future product for MQ in 5 days flat
-        specific:
-          client: 'MQ: Transforming Mental Health'
-          date: 26 Sep 2018
-          read_time: '3'
-          tags:
-            - name: Digital products
-              slug: digital-products
-            - name: Wagtail
-              slug: wagtail
-      - title: Multi-awarding winning site for one of the world's leading universities
-        specific:
-          client: University of Oxford
-          date: 19 Feb 2016
-          read_time: '4'
-          tags:
-            - name: Digital products
-              slug: digital-products
-      - title: A PWA in 4 weeks for Mozilla (oh yeah!)
-        specific:
-          client: Mozilla
-          date: 5 Dec 2018
-          read_time: '4'
-          tags:
-            - name: Digital products
-              slug: digital-products
+  work_pages:
+    - title: Imagining and designing a future product for MQ in 5 days flat
+      client: 'MQ: Transforming Mental Health'
+      date: 26 Sep 2018
+      read_time: '3'
+      tags:
+        - name: Digital products
+          slug: digital-products
+        - name: Wagtail
+          slug: wagtail
+    - title: Multi-awarding winning site for one of the world's leading universities
+      client: University of Oxford
+      date: 19 Feb 2016
+      read_time: '4'
+      tags:
+        - name: Digital products
+          slug: digital-products
+    - title: A PWA in 4 weeks for Mozilla (oh yeah!)
+      client: Mozilla
+      date: 5 Dec 2018
+      read_time: '4'
+      tags:
+        - name: Digital products
+          slug: digital-products
 
 tags:
   pageurl:
```

Comment:
The pageurl needs updating here to match the change in the template (ends up with a double // in the pattern library otherwise):

![Screenshot 2024-04-30 at 12 08 11](https://github.com/torchbox/torchbox.com/assets/771869/dc6e7ee8-a4da-4931-b4e4-38fda319863e)